docs/app/pages/components/components-sections/wizard/marquee-wizard/marquee-wizard.component.html

```diff
@@ -311,6 +311,17 @@ <h1 class="m-t-nil">Sixth step title</h1>
     <tr uxd-api-property name="gutterSize" type="number" defaultValue="10">
         Width of the splitter in pixels. This is only accessible when resizable is enabled.
     </tr>
+    <tr uxd-api-property name="footerTemplate" type="TemplateRef">
```

Comment:
Could we provide the interface information for the context `TemplateRef<INTERFACE HERE>`  so it is evident which type to import.